containers/Makefile

```diff
@@ -35,7 +35,6 @@ remote_containers := ncbi-amrfinderplus.sif \
 					mykrobe.sif \
 					nanoplot.sif \
 					perl-json.sif \
-					prodigal.sif \
```

Comment:
Is this removal mentioned in the CHANGELOG?

Reply:
It is now!